src/sentry/templates/sentry/integrations/gitlab-config.html

```diff
@@ -0,0 +1,32 @@
+{% extends "sentry/bases/modal.html" %}
+{% load crispy_forms_tags %}
+{% load sentry_assets %}
+{% load i18n %}
+
+{% block wrapperclass %} narrow auth {% endblock %}
+{% block modal_header_signout %} {% endblock %}
+
+{% block title %} {% trans "Gitlab Setup" %} | {{ block.super }} {% endblock %}
+
+{% block main %}
+<h3>Gitlab Configuration</h3>
+
+<form action="" method="post" class="form-stacked">
+  {% csrf_token %}
+  <input type="hidden" name="provider" value="vsts" />
```

Comment:
this has `'vsts'` as the provider

Reply:
Honestly, we could probably just extend the vsts template (create a base one and give it input), but it really isn't that bad.

Reply:
also it's funny cause I copy/pasta this from https://github.com/getsentry/sentry/blob/master/src/sentry/templates/sentry/integrations/github-enterprise-config.html which also has `vsts` 🙄